src/components/common-components/drawer/jira-like-drawer.component.tsx

```diff
@@ -10,7 +10,7 @@ import ScssVars from "../../../assets/styles/styles/custom/_variables.module.scs
 
 const useStyles = makeStyles({
   drawer: {
-    minWidth: 690,
+    width: 690,
```

Comment:
Teraz drawer będzie się trzymał w miejscu aka nie rozszerzał kiedy pojawią się dłuższe komunikaty błędów. Pytanie czy na dłuższą metę to jest dobre rozwiązanie

Reply:
Mi się wydaje, że my powinniśmy przekazywać szerokość do drawera przy jego otwarciu zamiast definiować to w css.

@maciekb05, co ty na to?